pipeline.py

```diff
@@ -358,8 +356,11 @@ def main(argv: Sequence[str]) -> None:
 
   with open(dest_path, "r") as f:
     js = json.load(f)
-    for _, v in js["pipelineSpec"]["deploymentSpec"]["executors"].items():
-      v["container"]["image"] = f"{v['container']['image']}:{FLAGS.image_tag}"
+    for k, v in js["pipelineSpec"]["deploymentSpec"]["executors"].items():
+      if k == "exec-convert":
+        v["container"]["image"] = f"{v['container']['image']}:{_gpu_to_dsm(config['deploy_gpu_type'])}"
```

Comment:
One thing that comes to mind now is that, even though it's ok to skip building the converter images in the daily build, I think we should still qualify them before we release them. So we should be able to reference different versions of the images. This will help if we wanted to fix them in the future without endangering breaking the released version.

Reply:
Agreed.

This makes sense to roll into the other suggestion of having a single convert image.

Reply:
Done.